src/Round.js

```diff
@@ -0,0 +1,34 @@
+const Turn = require("./Turn");
+
+class Round {
+  constructor(deck) {
+    this.deck = deck;
+    this.turns = 0;
+    this.incorrectGuesses = [];
+  }
+
+  returnCurrentCard() {
+    return this.deck.cards[0];
+  }
+
+  takeTurn(guess) {
+    this.turns ++;
+    let newTurn = new Turn(guess, this.returnCurrentCard());
+    newTurn.id = this.turns;
+    this.deck.cards.shift();
+    if (!newTurn.evaluateGuess()) {
+      this.incorrectGuesses.push(newTurn.id);
+    }
+    return newTurn.giveFeedback();
+  }
+
+  calculatePercentCorrect() {
+    return Math.floor((100 - (this.incorrectGuesses.length / this.turns * 100)));
```

Comment:
This line is a little long, but the math holds up. I will be thinking about how to refactor this in order to accomplish the same logic in fewer characters.